src/components/MissionsComponent.jsx

```diff
@@ -0,0 +1,61 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { joinMission } from '../redux/missions/missionsSlice';
+import '../css/MissionsComponent.css';
+
+const MissionsComponent = () => {
+  const dispatch = useDispatch();
+  const { missions } = useSelector((state) => state.missions);
+
+  return (
+    <main>
+      <table>
+        <thead>
+          <tr>
+            <th>Mission</th>
+            <th>Description</th>
+            <th>Status</th>
+            <th className="empty-space">
+          &nbsp;
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {
+            missions.map((mission) => {
+              const {
+                name, id, description, joined,
+              } = mission;
+              return (
+                <tr key={id} data-testid="mission">
+                  <th>{name}</th>
+                  <td>
+                    {description}
+                  </td>
+                  <td>
+                    <button
+                      type="button"
+                      className={`${joined ? 'active' : 'non-active'} member-button`}
+                    >
+                      {joined ? 'Active Member' : 'Not a Member'}
+                    </button>
+                  </td>
+                  <td>
+                    <button
+                      type="button"
+                      className={`${joined ? 'leave-mission' : 'join-mission'} mission-button`}
+                      onClick={() => dispatch(joinMission({ id }))}
+                    >
+                      { joined ? 'Leave Mission' : 'Join Mission'}
+                    </button>
+                  </td>
+                </tr>
+              );
+            })
+          }
+        </tbody>
+      </table>
```

Comment:
- [ ] The missions table is not rendering  as you can see in the caption below, kindly fix the issue:


<img width="1355" alt="Screenshot 2023-10-13 at 8 07 19 PM" src="https://github.com/rmiyoyo/space-travelers-hub/assets/53347966/4d69aa57-d560-4303-975e-3181b9bbedfd">